src/options.js

```diff
@@ -20,38 +20,205 @@ if (typeof window !== "undefined" && typeof window.define !== "undefined") {
   window.define("@jupyter-widgets/output", output);
 }
 
+/**
+ * Thebe options object
+ */
+export class Options {
```

Comment:
In order to document the options object in this way using jsdoc, we need to move away from the object literal and create an object class. This is potentially a good step as this is where we could also do some validation against a schema for the options. 

However, it also means that we are documenting options in. a few different places now, including in the main readme. It feels good to go with jsdoc generated docs for the options, as then the docs are generated from code as it is added / changed, so perhaps use this as the way to document the options and update the readme etc.. to point here.

Reply:
As a general rule, I am fine with having a single source of truth for documentation and then simply referring to that. If our README is getting complex enough, we should cut it down to minimal information needed for people to get started, and then push the full docs into our website via jsdoc.